.github/workflows/registrar-image.yaml

```diff
@@ -0,0 +1,44 @@
+name: Build and Publish kcp-registrar Image
+
+on:
+  push:
+    branches:
```

Comment:
Could you add [path filtering](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths)?
A new build is only required when something is modified in images/kcp-registrar

Reply:
I'm on it.